.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Werden die style auch von VSCode gelesen? 
Ansonsten macht es glaube ich nicht wirklich Sinn, weil dann trotzdem jeder für sich selbst formatiert. 
Dafür gibt es das `.editorconfig` format, welches von mehreren IDEs unterstützt wird.

Reply:
Das hab ich da nur reingepackt, damit mir CLion nicht mit den standardmäßigen 4-Spaces-Indentation alles umformatiert 😉.

Eine globale .editorconfig wäre da auf jeden Fall besser, da ich auch nicht genau weiß ob VS Code erkennt, dass die Dateien mit 2 Leerzeichen geschrieben wurden.

Reply:
mit dem richtigen PLugin auf jeden fall.

Reply:
Ich bin definitiv auch für eine vernünftige `.editorconfig`. Das Plugin entsprechend  in der README als Requirement nennen